api/graphql/schema.graphql

```diff
@@ -738,7 +742,8 @@ type Query {
     poolByKey(key: String! @eq): Pool @find
     pools: [Pool]! @all
     poolCandidate(id: ID! @eq): PoolCandidate @find @guard @can(ability: "view", query: true)
-    poolCandidates: [PoolCandidate]! @all(scopes: ["expiryFilter"]) @guard @can(ability: "viewAny")
+    poolCandidates(includeIds: [ID] @in(key: "id")): [PoolCandidate]! @all(scopes: ["expiryFilter"]) @guard @can(ability: "viewAny")
```

Comment:
something that comes to mind, we will probably want to remove the expiry scope from this as there will be toggled filters that allow viewing of expired candidates say for printing purposes
if the scope remains you get the oddity of being able to select the expired candidate due to the poolCandidatesPaginated query but unable to print or do anything due to the poolCandidates query auto-removing expired

Reply:
I've now removed the scope from the query.